src/main/java/frc/robot/OI.java

```diff
@@ -8,13 +8,18 @@
 public class OI {
     //Operator Interface (OI) class containing all control information
     private static final int kDriverJoystickPort = 0;
+    private static final int kTeleopJoystickPort = 1;
```

Comment:
What is the teleop joystick? Do you mean operator joystick? If so, all operator joystick bindings should be in the Operator class, not Driver